src/shared/__tests__/visual/community.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { test } from '@playwright/test';
+import { takeScreenshot } from './utils';
+
+test('Community page', async ({ page }) => {
+  await page.goto('/community');
```

Comment:
```suggestion
  await page.goto(ROUTES.COMMUNITY);
```

Reply:
Done, but it adds some new warnings 🙂
![image](https://github.com/user-attachments/assets/39497bc0-0a66-4294-8005-043ce48fb2f6)

Reply:
we're going to refactor `app` folder soon and will move all the constants to `shared`